src/main/java/uk/gov/pay/ledger/report/dao/PerformanceReportDao.java

```diff
@@ -41,4 +60,13 @@ public PerformanceReportEntity performanceReportForPaymentTransactions(Performan
             return query.map(new PerformanceReportEntityMapper()).findOnly();
         });
     }
+
+    public List<GatewayAccountMonthlyPerformanceReportEntity> monthlyPerformanceReportForGatewayAccounts(ZonedDateTime startDate, ZonedDateTime endDate) {
```

Comment:
why is this returning a list? shouldn't it just be a single object that contains the statistics for the month?

Reply:
No. The GatewayAccountMonthlyPerformanceReportEntity is an object that represents the average amount, total volume etc... for a particular gateway_account_id for that month. So the List<...> represents the average amount, total volume for each gateway_account_id in the db.

Reply:
ah, got that. 👍